games-simulation/simutrans/simutrans-120.2.0.recipe

```diff
@@ -0,0 +1,82 @@
+SUMMARY="A transport simulation game"
+DESCRIPTION="Simutrans is an open-source transportation simulation game \
+available for Windows, Mac, Linux and BeOS. Your goal is to establish a \
+successful transport company and build the necessary infrastructure to \
+transport passengers, mail and goods between cities, factories and \
+touristic attractions by land, air and water."
+HOMEPAGE="http://www.simutrans.com"
+SvnRevision="8077"
+GitRevision="ea08f1eed3fd8ffdd017a522898450047f83d768"
+SOURCE_URI="https://github.com/aburch/simutrans/archive/$GitRevision.tar.gz"
+CHECKSUM_SHA256="96dca621cde48466b38fa6cf7116e18c72d8c6cadca6733cfdc1f2a4048d2e6b"
+SOURCE_FILENAME="simutrans-$portVersion.tar.gz"
+SOURCE_DIR="simutrans-$GitRevision"
+COPYRIGHT="1997-2004 Hj. Malthaner
+	2005-2017 The Simutrans Team"
+LICENSE="Artistic"
+REVISION="1"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+PROVIDES="
+	simutrans$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libsdl$secondaryArchSuffix
+	lib:libsdl_mixer$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	simutrans_pakset$secondaryArchSuffix >= 120.2
+	timgmsoundfont
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libsdl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libbz2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:g++$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:git
+	cmd:svn
+	cmd:curl
```

Comment:
I've uploaded simutrans-120.2.1~test.recipe. It downloads the source code twice, once as tarball from the git mirror (SOURCE_URI) and once as "official" zip (incl. the language files) from sourceforge (SOURCE_URI_2). Since I prefer the configure mechanism of the git mirror (you get at least a binary that can actually run scenarios), the sourceforge zip is only used for supplementing the language files so that svn, curl, git and get_lang_files.sh can be omitted.
If you prefer this solution I'll rename it to simutrans-120.2.1.recipe and will remove the other recipes.